easybuild/easyblocks/m/molpro.py

```diff
@@ -0,0 +1,194 @@
+            self.cfg.update('configopts', "-%s -%s" % (os.environ['CC_SEQ'], os.environ['F90_SEQ']))
+            if 'MPI_INC_DIR' in os.environ:
+                self.cfg.update('configopts', "-mpp -mppbase %s" % os.environ['MPI_INC_DIR'])
+            else:
+                raise EasyBuildError("$MPI_INC_DIR not defined")
+        else:
+            self.cfg.update('configopts', "-%s -%s" % (os.environ['CC'], os.environ['F90']))
+
+        # BLAS/LAPACK
+        if 'BLAS_LIB_DIR' in os.environ:
+            self.cfg.update('configopts', "-blas -blaspath %s" % os.environ['BLAS_LIB_DIR'])
+        else:
+            raise EasyBuildError("$BLAS_LIB_DIR not defined")
+
+        if 'LAPACK_LIB_DIR' in os.environ:
+            self.cfg.update('configopts', "-lapack -lapackpath %s" % os.environ['LAPACK_LIB_DIR'])
+        else:
+            raise EasyBuildError("$LAPACK_LIB_DIR not defined")
+
+        # 32 vs 64 bit
+        if self.toolchain.options.get('32bit', None):
+            self.cfg.update('configopts', '-i4')
+        else:
+            self.cfg.update('configopts', '-i8')
+
+        run_cmd("./configure -batch %s" % self.cfg['configopts'])
+
+        cfgfile = os.path.join(self.cfg['start_dir'], 'CONFIG')
+        cfgtxt = read_file(cfgfile)
+
+        # determine original LAUNCHER value
+        launcher_regex = re.compile('^LAUNCHER=(.*)$', re.M)
+        res = launcher_regex.search(cfgtxt)
+        if res:
+            self.orig_launcher = res.group(1)
+            self.log.debug("Found original value for LAUNCHER: %s", self.orig_launcher)
+        else:
+            raise EasyBuildError("Failed to determine LAUNCHER value")
+
+        # determine full installation prefix
+        prefix_regex = re.compile('^PREFIX=(.*)$', re.M)
+        res = prefix_regex.search(cfgtxt)
+        if res:
+            self.full_prefix = res.group(1)
+            self.log.debug("Found full installation prefix: %s", self.full_prefix)
+        else:
+            raise EasyBuildError("Failed to determine full installation prefix")
+
+        # determine MPI launcher command that can be used during build/test
+        # obtain command with specific number of cores (required by mpi_cmd_for), then replace that number with '%n'
+        launcher = self.toolchain.mpi_cmd_for('%x', self.cfg['parallel'])
+        launcher = launcher.replace(' %s' % self.cfg['parallel'], ' %n')
+
+        # patch CONFIG file to change LAUNCHER definition, in order to avoid having to start mpd
+        for line in fileinput.input(cfgfile, inplace=1, backup='.orig'):
+            line = re.sub(r"^(LAUNCHER\s*=\s*).*$", r"\1 %s" % launcher, line)
+            sys.stdout.write(line)
+
+        # reread CONFIG and log contents
+        cfgtxt = read_file(cfgfile)
+        self.log.info("Contents of CONFIG file:\n%s", cfgtxt)
+
+        # make sure license token is available
+        token_path = os.path.join(os.environ['HOME'], '.molpro', 'token')
```

Comment:
ah, yes, this check can be removed, since we made sure it has been put in place at the start of `configure_step`